homeassistant/components/iqvia/__init__.py

```diff
@@ -75,6 +75,14 @@
 )
 
 
+@callback
+def async_get_api_category(sensor_type):
+    """Return the API category that a particular sensor type should use."""
+    if sensor_type in API_CATEGORY_MAPPING:
+        return API_CATEGORY_MAPPING[sensor_type]
+    return sensor_type
```

Comment:
You can pass a default value to `.get` which is returned if key not found:

```suggestion
    return API_CATEGORY_MAPPING.get(sensor_type, sensor_type)
```